app.js

```diff
@@ -0,0 +1,12 @@
+const { server } = require('./server');
+const mongoose = require('mongoose');
+const port = process.env.PORT || 5000;
+
+mongoose.Promise = global.Promise;
+mongoose.connect('mongodb://lambdauser1:lambdauser1@ds239439.mlab.com:39439/notes', {
+  useMongoClient: true
+});
```

Comment:
Glad to see you making use of MLab. Would be good to add error handling in case the MongoDB is not available